src/default-reducers.js

```diff
@@ -94,53 +127,53 @@ export function retrieveManyAborted(state, action) {
   };
 }
 
-export function retrieveManyResetResolution(state, action) {
+export function retrieveManyResetResolution(idAttr, state, action) {
   return {
     ...state,
     resourcesListMeta: {
-      retrievingStatus: null
+      retrievingStatus: resourceStatuses.NULL
```

Comment:
We synced up about the use case for this. It'll be documented when the time comes, so I'm not going to elaborate on ithere.